ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -1004,12 +1006,19 @@ def _on_span_started(span: Span):
                 span.set_tag_str("langchain.request.%s.parameters.%s.%s" % (llm_provider, param, k), str(v))
 
     def _on_span_finished(span: Span, streamed_chunks):
-        if span.error or not integration.is_pc_sampled_span(span):
+        joined_chunks = streamed_chunks[0]
+        for chunk in streamed_chunks[1:]:
+            joined_chunks += chunk  # base message types support __add__ for concatenation
```

Comment:
i think it would involve typing it with `langchain_core`. while `langchain_core` _should_ be a defined module if this function is being traced, i always like to preface any use if `langchain_core`, etc. by checking if it exists first, which I'm not sure how to do with a type annotation. open to a suggestion though!

Reply:
Hmm that makes sense, I think the comment here suffices then - thanks!